samples/ig-dashboard/package.json

```diff
@@ -1,22 +1,34 @@
 {
-	"name": "ig-dashboard",
-	"version": "1.0.0",
-	"private": true,
-	"dependencies": {
-		"@infragistics/ignite-ui-full": "^18.1.152",
-		"babel-runtime": "6.22.0",
-		"bootstrap": "^3.3.7",
-		"igniteui-react": "^1.1.0",
-		"jquery": ">=3.5.0",
-		"jquery-ui": ">=1.13.0",
-		"react": "^16.6.3",
-		"react-dom": "^16.6.3",
-		"react-scripts": "1.0.17"
-	},
-	"scripts": {
-		"start": "react-scripts start",
-		"build": "react-scripts build",
-		"test": "react-scripts test --env=jsdom",
-		"eject": "react-scripts eject"
-	}
+  "name": "ig-dashboard",
+  "version": "1.0.0",
+  "private": true,
+  "dependencies": {
+    "@infragistics/ignite-ui-full": "^18.1.152",
```

Comment:
It's a good idea to update the ignite-ui-full version as well. The latest versions is 21.2.9.

Reply:
This applies for all of the samples, I guess